composites/Plugin/Shared/components/SynonymsInput.js

```diff
@@ -16,9 +12,9 @@ const SynonymsInput = ( { id, label, value, onChange, explanationText } ) => {
 				{ label }
 			</YoastInputLabel>
 			{ explanationText !== null && (
```

Comment:
not sure, why not simply use `{ explanationText && (`

Reply:
not sure either, changing it.